projects/core/src/cart/store/actions/cart-voucher.action.spec.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  entityFailMeta,
+  entityLoadMeta,
+  entitySuccessMeta,
+} from '../../../state/utils/entity-loader/entity-loader.action';
+import {
+  failMeta,
+  loadMeta,
+  successMeta,
+} from '../../../state/utils/loader/loader.action';
+import { ADD_VOUCHER_PROCESS_ID, CART_DATA } from '../cart-state';
+import * as fromCartVoucher from './cart-voucher.action';
+
+const userId = 'xxx@xxx.xxx';
+const cartId = 'testCartId';
+const voucherId = 'testVoucherId';
+
+describe('Cart-voucher Actions', () => {
+  describe('AddCartVoucher Actions', () => {
```

Comment:
Why using so many "describe"s ?

Reply:
it has levels as cart-voucher-> addCartVoucher Actions -> add/add success/add fail